lib/routes/account.js

```diff
@@ -214,6 +214,7 @@ module.exports = function (
               emailVerified: account.emailVerified,
               verifierSetAt: account.verifierSetAt,
               createdAt: parseInt(query._createdAt),
+              wantedKeys: requestHelper.wantsKeys(request),
```

Comment:
Correct.  I think a better name here would be `mustVerify`, which would make it more obvious that this has no effect outside of the signin-confirmation feature.